src/main.cpp

```diff
@@ -48,10 +50,10 @@ setup () {
 
 void
 loop () {
-    // haptics_test();
-    compass.compass_update();
+    run_haptics(bach, 32, 10);
```

Comment:
When reading a function call in the middle of main with various parameters like this, it can be a little hard to know what the numbers mean without checking the definition. I recommend using inline comments like this:
```
run_haptics(bach, \*num_pulses=*\32, \*num_cycles=*\10);
```
For objects like the pattern, it's not as much as a concern, but for loose numbers, it is nice.